crates/ironrdp-connector/Cargo.toml

```diff
@@ -24,13 +24,13 @@ ironrdp-core = { path = "../ironrdp-core", version = "0.1" } # public
 ironrdp-error = { path = "../ironrdp-error", version = "0.1" } # public
 ironrdp-pdu = { path = "../ironrdp-pdu", version = "0.5", features = ["std"] } # public
 arbitrary = { version = "1", features = ["derive"], optional = true } # public
-sspi = "0.15" # public
+sspi = { path = "../../../sspi-rs" } # public
```

Comment:
https://github.com/Devolutions/sspi-rs/pull/457

Reply:
Note that we can’t publish until the dev branch is merged onto master.

Reply:
Version 0.16 is published!